planner/cascades/testdata/transformation_rules_suite_in.json

```diff
@@ -10,7 +10,8 @@
       "select a, max(b) from t group by a having a > 1",
       "select a, avg(b) from t group by a having a > 1 and max(b) > 10",
       "select t1.a, t1.b, t2.b from t t1, t t2 where t1.a = t2.a and t2.b = t1.b and t1.a > 10 and t2.b > 10 and t1.a > t2.b",
-      "select t1.a, t1.b from t t1, t t2 where t1.a = t2.a and t1.a = 10 and t2.a = 5"
+      "select t1.a, t1.b from t t1, t t2 where t1.a = t2.a and t1.a = 10 and t2.a = 5",
+      "select a, b from ((select a, b from t where a > 1) union all (select a, b from t where b > 5)) as t1"
```

Comment:
This test case won't trigger the rule `PushSelDownUnionAll`. Try `select a, b from ((select a, b from t) union all(select c as a, d as b from t)) as t1 where a > 1`